File: etc/test/install_redirect_test.py
import sys
import urllib2

URL = "http://dot.b4b4r07.com"
url = urllib2.urlopen(URL).geturl()

if not url == "https://raw.githubusercontent.com/b4b4r07/dotfiles/master/etc/install":
    sys.exit(1)

print 'ok: %s <-> %s' % (URL, url)

File: etc/test/install_redirect_test.pl
#!/usr/bin/perl

use strict;
use LWP::UserAgent;
use Data::Dumper;
use Test::More;

my $redirect = "https?:\/\/raw\.github.*\.com\/b4b4r07\/dotfiles\/master\/etc\/install";
my $url = 'http://dot.b4b4r07.com';

my $ua = LWP::UserAgent->new();
my $response = $ua->get($url);
my $uri = $response->request->uri;

like "$uri", qr/$redirect/, "$uri -> $url";
done_testing;
